fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/OCFLPersistentStorageSession.java

```diff
@@ -153,12 +189,37 @@ public InputStream getBinaryContent(final String identifier, final Instant versi
     }
 
     @Override
-    public void commit() throws PersistentStorageException {
+    public void commit(final CommitOption option) throws PersistentStorageException {
```

Comment:
Do we want to implement per object versioning down the road?  I thought we were going to punt in the short term.  Either way,  I would suggest we create a separate ticket and address later. Unless you think that will cause more pain than it is worth.

Reply:
If we are punting, it would probably be easier to punt by having the interface line up with the eventual goal. So that could mean leaving this parameter out here and then passing a default commit option into all of the object sessions. So everything might commit to a new version or mutable head for now. Otherwise, the clients calling this will either have to try to figure out what to pass, in which we'll remove later, or they'll all have to pick a default, in which case it'd be easier to centralize that here.

Reply:
How will users of OCFLPersistentStorageSession indicate that they want to override the default behavior in a particular  invocation?  Namely, imagine the default commit behavior is UNVERSIONED, but the user desires to mint a version?  Do we need a separate method? 
commitVersion()?

Reply:
> bbpennel:waffle: 2:52 PM
@dbernstein for an object that does not have autoversioning turned on, you would issue a request to create a memento, which would probably go through a CreateVersionPersister or something like that, which would call objectSession.commit(NEW_VERSION);
Its a bit awkward since we aren’t allowing creating mementos as part of a transactions since it needs to immediately commit the object session, which is a bit unusual
i don’t know that there’s a use case or API means of telling something with autoversioning to not version

Reply:
Hmmm.  looking back at this issue with committing versions I'm not sure I like it.
I'm going to hold off on changing that  for now in my PR.  I think it needs more discussion.